recipes/data-management/quantumleap/insert.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
for the other tools i don't think we provide sample code to test them. so probably this should be removed

Reply:
We do have, see for orion: https://github.com/smartsdk/smartsdk-recipes/tree/master/recipes/data-management/context-broker.

I added those because I found them useful in the original Orion documentation, which provides some snippets to try.